components/Chat/SendMessage.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+export default  function SendMessage({ handleSubmit, handleMessageChange, message }) {
```

Comment:
The distance between "default" and "function" looks to big, does prettier work correctly on saving the file?

Reply:
Is done, I work with phpstorm, there the prettier works sometimes a little bit different to vscode